src/mappings/geyser.ts

```diff
@@ -275,7 +291,22 @@ export function handleRewardsUnlocked(event: RewardsUnlocked): void {
 
 
 export function handleRewardsExpired(event: RewardsExpired): void {
-  // todo
+  let geyser = Geyser.load(event.address.toHexString());
+  let rewardToken = Token.load(geyser.rewardToken);
+  let amount = integerToDecimal(event.params.amount, rewardToken.decimals);
+
+  for (let i = 0; i < geyser.fundings.length; i++) {
+    let fundingId = (geyser.fundings as string[])[i];
+    let funding = Funding.load(fundingId);
+
+    // remove expired funding
+    if (funding.start.equals(event.params.start)
+    && funding.end.equals(funding.start.plus(event.params.duration))
+    && funding.amount.equals(amount)) {
+      store.remove('Funding', fundingId);
```

Comment:
do we also need to remove the item from the `geyser.fundings` list?

Reply:
actually, on second thought, would it be worth keeping the object around in the subgraph? it's pretty low volume relative to other schemas like the position which we are removing, and could be very useful to the creator to understand the history of the geyser

instead, we could mark it as stale or expired

Reply:
Removing it from the store automatically removes it from the `geyser.fundings` list so we're good there. 

Per our discussion offline, we're going to remove the funding object on expiration, at least for now.